root/static/scripts/release-editor/actions.js

```diff
@@ -69,6 +75,83 @@ const actions = {
     }));
   },
 
+  duplicateReleaseGroups: ko.observableArray([]),
```

Comment:
I moved these to where they're set and used, but please let me know if keeping them in `init.js` would be preferable.